### DISCUSSION TITLE
Problem with building android app

### DISCUSSION THREAD

**Discussion by SomeoneSK** [anyproto/contributors#39] [Q&A]
I'm having trouble with building the android app in android studio. When I try to build the app I get this error:

`Unable to make private void java.util.concurrent.CopyOnWriteArrayList.writeObject(java.io.ObjectOutputStream) throws java.io.IOException 
 accessible: module java.base does not "opens java.util.concurrent" to unnamed module @3ef07f53`

I can run the sample config but when I try to run config of the actual app it gives me this error.

I am on WIndows 11.

What am I doing wrong? Do I need to add any additional info?

**Comment by fuksman** [comment]
@anyproto/android

**Comment by uburoiubu** [comment]
@SomeoneSK, thanks for reaching out.

please provide additional info:

1) Android Studio version

2) your branch name

3) Your project structure:

File -> Project Structure -> Select "app" module:

<img width="794" alt="Screenshot 2023-07-26 at 14 38 15" src="https://github.com/anyproto/contributors/assets/21975311/085f7b48-ff06-4711-a63a-9604626c235f">

4) Are you trying to build your debug build?

**Reply by SomeoneSK** [reply]
1. Android Studio Giraffe | 2022.3.1
2. I just cloned the main repo branch to play with it first. 
The only thing that I changed is line 102 in file _build.gradle_:
`commandLine 'sh', '-c', 'git config core.hooksPath .githooks'`
to
`commandLine 'cmd', '/c', 'git config core.hooksPath .githooks'`
because when I tried to build it I was getting this error:
`A problem occurred starting process 'command 'sh''`
I also added _github.properties_ and _apikeys.properties_ files as [guide on readme](https://github.com/anyproto/anytype-kotlin#build-project) instructed.
3. My project structure: 
<img width="500" alt="image" src="https://github.com/anyproto/contributors/assets/53940106/a6cc19e7-2049-456b-8e26-c70f13e1a096">

4. Nope. I just cloned repo, created 2 files, changed 1 file and clicked build button.


Thank you

**Reply by SomeoneSK** [reply]
I still haven't solved this. If you have any idea, please share it with me. 
However, it is not very important so you don't need to waste too much time trying to help me.

**Reply by BrayanDSO** [reply]
Got the same issue. Solved after adding a github personal access key, as oriented in the README. Maybe the key isn't necessary anymore after some recent changes

**Reply by SomeoneSK** [reply]
Yeah, I did that already and it did not work. Thanks for trying to help though.
